blueprints/addon/files/.travis.yml

```diff
@@ -38,17 +38,16 @@ jobs:
     - stage: "Tests"
       name: "Tests"
       script:
-        - <% if (yarn) { %>yarn<% } else { %>npm run<% } %> lint:hbs
-        - <% if (yarn) { %>yarn<% } else { %>npm run<% } %> lint:js
-        - <% if (yarn) { %>yarn<% } else { %>npm<% } %> test
+        - <% if (yarn) { %>yarn lint<% } else { %>npm run lint<% } %>
```

Comment:
Can you bring back the combined suffix here please?

Reply:
Yep, absolutely! Done...